packages/compass-components/src/hooks/use-confirmation.spec.tsx

```diff
@@ -74,10 +74,10 @@ describe('use-confirmation', function () {
     });
   });
 
-  context('showConfirmation global function', function () {
+  context.only('showConfirmation global function', function () {
```

Comment:
Getting a
```
Uncaught TypeError: Failed to execute 'dispatchEvent' on 'EventTarget': parameter 1 is not of type 'Event'.
```
Error on confirmation modal renders. Need to sort that out.

Reply:
Done, had to add a mock to the jsdom mocha config though. Would be nice to remove it eventually, haven't found an issue we can track there yet.